bridge/gitlab/export.go

```diff
@@ -159,7 +159,7 @@ func (ge *gitlabExporter) exportBug(ctx context.Context, b *cache.BugCache, sinc
 	gitlabID, ok := snapshot.GetCreateMetadata(metaKeyGitlabId)
 	if ok {
 		gitlabBaseUrl, ok := snapshot.GetCreateMetadata(metaKeyGitlabBaseUrl)
-		if ok && gitlabBaseUrl != ge.conf[gitlabBaseUrl] {
+		if ok && gitlabBaseUrl != ge.conf[keyGitlabBaseUrl] {
```

Comment:
fix is here

Reply:
Nice catch, this can definitely have stopped git-bug from exporting bugs